Gems/Atom/Feature/Common/Code/Source/SplashScreen/SplashScreenPass.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Memory/SystemAllocator.h>
+
+#include <AzCore/Component/TickBus.h>
+#include <Atom/RPI.Public/Pass/FullscreenTrianglePass.h>
+#include <Atom/RPI.Reflect/Pass/PassDescriptor.h>
+#include <Atom/Feature/SplashScreen/SplashScreenSettings.h>
+
+namespace AZ::Render
+{
+    class SplashScreenPass
+        : public AZ::RPI::FullscreenTrianglePass
+        , public AZ::TickBus::Handler
+    {
+        AZ_RPI_PASS(SplashScreenPass);
+
+    public:
+        AZ_RTTI(AZ::Render::SplashScreenPass, "{B12F4E30-94ED-4F69-A17D-85C65853ACD9}", AZ::RPI::FullscreenTrianglePass);
+        AZ_CLASS_ALLOCATOR(AZ::Render::SplashScreenPass, AZ::SystemAllocator, 0);
+        virtual ~SplashScreenPass();
+
+        static AZ::RPI::Ptr<SplashScreenPass> Create(const AZ::RPI::PassDescriptor& descriptor);
+
+        // TickBus overrides...
+        void OnTick(float deltaTime, AZ::ScriptTimePoint time) override;
+
+        // Scope producer functions...
+        void SetupFrameGraphDependencies(AZ::RHI::FrameGraphInterface frameGraph) override;
```

Comment:
Comments for the indented use of public methods. Helpfully to call out if game developers or engine developers would need to use this API.